pandas/core/frame.py

```diff
@@ -2949,9 +2949,71 @@ def f(vals):
     #----------------------------------------------------------------------
     # Sorting
 
+    @Appender(_shared_docs['sort_values'] % _shared_doc_kwargs)
+    def sort_values(self, by, axis=0, ascending=True, inplace=False,
+               kind='quicksort', na_position='last'):
+
+        axis = self._get_axis_number(axis)
+        labels = self._get_axis(axis)
+
+        if axis != 0:
+            raise ValueError('When sorting by column, axis must be 0 '
+                             '(rows)')
```

Comment:
yeh this is just not implemented. not sure why. I'll put a bullet and see if i can do it. Was trying not to change any impl.